### PR TITLE
Add questions for optimistic rollup and bridges

### PR DESCRIPTION
**Pull request by gregerodrigues**

### CODE REVIEW

File: src/bridges-deep-dive-dodao-academy.yaml [hunk 1]
```diff
@@ -90,18 +91,21 @@ steps:
...
             key: 6db3906f
-          - content: C
+          - content: Executors
             key: 3975ac56
-          - content: D
+          - content: Validators
             key: bdf62e57
-        content: Q1
-        type: SingleChoice
+        content:  What type of workers are involved in assessing the security of a validating bridge?
```
Comment (RobinNagpal) [inline src/bridges-deep-dive-dodao-academy.yaml:107]: are `Sequencers` and `Executors` responsible for validation?

File: src/optimistic-rollups-dodao-academy.yaml [hunk 2]
```diff
@@ -75,17 +75,19 @@ steps:
...
             key: 9228635b
-          - content: C
+          - content: Instant transactions
             key: 09206bad
-          - content: D
+          - content: Charging less gas fee
             key: 0be114b4
-        content: Q
-        type: SingleChoice
+        content: How did layer 1 benefit from the introduction of layer 2 frameworks?
```
Comment (RobinNagpal) [inline src/optimistic-rollups-dodao-academy.yaml:89]:
```suggestion
        content: How do we benefit from the introduction of layer 2 frameworks?
```

File: src/optimistic-rollups-dodao-academy.yaml [hunk 3]
```diff
@@ -303,20 +306,20 @@ steps:
...
+          - content: The state transition itself
             key: c038a5a7
-          - content: C
+          - content: Executor's asserted post-state
             key: f00d64fb
-          - content: D
+          - content: All of the above
             key: 9b0493fa
-        content: Q3
+        content:  The list of inclusion proofs in One-round fraud proof for quick and easy fraud detection are?
```
Comment (RobinNagpal) [inline src/optimistic-rollups-dodao-academy.yaml:319]: The question and option as not easy for anyone new to answer. HAs very specific language

File: src/bridges-deep-dive-dodao-academy.yaml (hunk 1 above)
Comment (gregerodrigues) [reply src/bridges-deep-dive-dodao-academy.yaml:107]: Sir, the question is not about validation, it's about assessing the security required for validation

**Pull request closed without merging**